src/Symfony/Component/Debug/ErrorHandler.php

```diff
@@ -436,11 +425,12 @@ public function handleError($type, $message, $file, $line)
 
             // Clean the trace by removing function arguments and the first frames added by the error handler itself.
             if ($throw || $this->tracedErrors & $type) {
-                $backtrace = $backtrace ?: $errorAsException->getTrace();
+                $backtrace = $errorAsException->getTrace();
                 $lightTrace = $this->cleanTrace($backtrace, $type, $file, $line, $throw);
                 $this->traceReflector->setValue($errorAsException, $lightTrace);
             } else {
                 $this->traceReflector->setValue($errorAsException, array());
+                $backtrace = array();
             }
         }
 
```

Comment:
I didn't removed them because the code is pretty much tricky and I'm not sure what it's used for so I didn't remove anything. ^^'